spec/inputs/tcp_spec.rb

```diff
@@ -402,29 +402,6 @@ def get_port
                 raise StandardError, "blah"
               end
             end
-
-            it "should log the error on accept" do
-              allow(input.logger).to receive(:error).with(any_args)
-              
-              stop = Thread.new {
```

Comment:
no sure I understand why we need to spawn a thread here?

Reply:
@colinsurprenant, that is one of the two integration tests that I deleted as part of this PR. I added some extra detail to the PR description to explain why those two tests were removed.

Reply:
uh ... wtf...  :( of course.